frontend/turbo.json

```diff
@@ -1,29 +1,16 @@
 {
     "$schema": "https://turborepo.org/schema.json",
     "globalEnv": [
-        "INTERNET_IDENTITY",
-        "NFID_URL",
         "DFX_NETWORK",
-        "IC_URL",
-        "II_DERIVATION_ORIGIN",
-        "CUSTOM_DOMAINS",
-        "STORAGE_INDEX_CANISTER",
-        "LEDGER_CANISTER_ICP",
```

Comment:
It seems to me that we could now change the value of one of the common env vars and turbo would _not_ invalidate the cache.

Reply:
It would count as an input file change, so it would still invalidate the cache, but only for the app build, which is exactly what we want.